### ISSUE TITLE
Cytoscape tutorial is outdated

### ISSUE THREAD

**Issue by AlexanderPico** [biocore/qiime#2172]
Your documentation on Making Cytoscape Networks is a bit outdated, making it difficult to follow with the latest UI and features in Cytoscape.

http://qiime.org/tutorials/making_cytoscape_networks.html

To easy the burden of updating, you might adapt the tutorial to reference tutorials.cytoscape.org and just point out the Quiime-specific parts.  Or you could contribute your own tutorial to this open platform and make it easy for others to keep it up-to-date.  Here is a general tutorial that covers many of the same steps for your reference:

http://opentutorials.cgl.ucsf.edu/index.php/Tutorial:Introduction_to_Cytoscape_3.1-part2

**Comment by gregcaporaso** [comment]
I agree: updating the current tutorial to reference the Cytoscape tutorials and noting that it's outdated would be a good change. For QIIME 2, it'll be better to integrate more closely with the Cytoscape tutorials page, as you suggest. Thanks @AlexanderPico!

**Comment by rob-knight** [comment]
Note that the cytoscape devs are in our building so it might be worth more detailed engagement to do something that’s cutting edge now vs what we did in 2008…

On Nov 15, 2016, at 2:33 PM, Greg Caporaso <notifications@github.com<mailto:notifications@github.com>> wrote:


I agree: updating the current tutorial to reference the Cytoscape tutorials and noting that it's outdated would be a good change. For QIIME 2, it'll be better to integrate more closely with the Cytoscape tutorials page, as you suggest. Thanks @AlexanderPico<https://github.com/AlexanderPico>!

**Comment by gregcaporaso** [comment]
> more detailed engagement to do something that’s cutting edge

Completely agree.
